content/general/concepts/ide/ide.md

```diff
@@ -15,7 +15,7 @@ CatalogContent:
   - 'paths/front-end-engineer-career-path'
 ---
 
-An IDE, or Integrated Development Environment, enables programmers to consolidate the different aspects of writing a computer program.
+An **IDE**, or Integrated Development Environment, enables programmers to consolidate the different aspects of writing a computer program.
 
 IDEs increase programmer productivity by combining common activities of writing software into a single application: editing source code, building executables, and debugging.
 
```

Comment:
For the header on the next line, I would suggest renaming it to "Common Features". Then, I would add a hashtag `#` to the following headers:

- ~Autocomplete~ (change this to "Autocompletion")
- Building Executables
- Debugging